src/views/OpenOrders.vue

```diff
@@ -0,0 +1,220 @@
+
+          <div class="order-tags">
+            <ion-chip outline>
+              <ion-icon :icon="pricetag" />
+              <ion-label>NN10584</ion-label>
+            </ion-chip>
+            <ion-button fill="clear" class="mobile-only" color="danger">
+              <ion-icon slot="icon-only" :icon="refreshCircleOutline" />
+            </ion-button>
+          </div>
+
+          <div class="order-metadata">
+            <ion-label>
+              Next Day Shipping
+              <p>Ordered 28th January 2020 2:32 PM EST</p>
+            </ion-label>
+          </div>
+        </div>
+
+        <div class="order-item">
+          <div class="product-info">
+            <ion-item lines="none">
+              <ion-thumbnail>
+                <img src="https://dev-resources.hotwax.io/resources/uploads/images/product/m/j/mj08-blue_main.jpg" />
+              </ion-thumbnail>
+              <ion-label>
+                <p class="overline">WJ06-XL-PURPLE</p>
+                Juno Jacket
+                <p>Blue XL</p>
+              </ion-label>
+            </ion-item>
+          </div>
+          <div class="product-metadata">
+            <ion-note>49 pieces in stock</ion-note>
+          </div>
+        </div>
+
+        <div class="actions">  
+          <div class="positive-action"></div>
+          <div class="negative-action">
+            <ion-button class="desktop-only" fill="outline" color="danger">Recycle</ion-button>
+          </div>
+        </div>
+      </ion-card>
+
+      <ion-fab class="mobile-only" vertical="bottom" horizontal="end" slot="fixed">
+        <ion-fab-button @click="assignPickers">
+          <ion-icon :icon="print" />
+        </ion-fab-button>
+      </ion-fab> 
+    </ion-content>
+  </ion-page>
+</template>
+
+<script lang="ts">
+import { IonButton, IonButtons, IonCard, IonChip, IonCheckbox, IonContent, IonFab, IonFabButton, IonHeader, IonLabel, IonIcon, IonItem, IonMenuButton, IonNote, IonPage, IonSearchbar, IonThumbnail, IonTitle, IonToolbar, modalController } from '@ionic/vue';
+import { defineComponent } from 'vue';
+import { options, pricetag, print, refreshCircleOutline } from 'ionicons/icons';
+import AssignPickerModal from '@/views/AssignPickerModal.vue';
+
+export default defineComponent({
+  name: 'OpenOrders',
+  components: {
+    IonButton,
+    IonButtons,  
+    IonCard,
+    IonChip,
+    IonCheckbox,
+    IonContent,
+    IonFab,
+    IonFabButton,
+    IonHeader,
+    IonLabel,
+    IonIcon,
+    IonItem,
+    IonMenuButton,
+    IonNote,
+    IonPage,
+    IonSearchbar,
+    IonThumbnail,
+    IonTitle,
+    IonToolbar
+  },
+  methods: {
+    async assignPickers() {
+      const bgjobmodal = await modalController.create({
+        component: AssignPickerModal
+      });
+      return bgjobmodal.present();
+    },
+  },
+  setup() {
+    return{
+      options,
+      pricetag,
+      print,
+      refreshCircleOutline
+    }
+  }
+});
+</script>
+
+<style scoped>
+
+.filters > ion-item {
```

Comment:
this should be a global class since we are adding this for all filters.

Reply:
Done